fieldservice/views/fsm_schedule.xml

```diff
@@ -0,0 +1,39 @@
+<odoo>
+
+    <!-- Field Service Schedule -->
+    <record id="fsm_schedule_tree_view" model="ir.ui.view">
+        <field name="name">Schedules</field>
+        <field name="model">resource.calendar</field>
+        <field name="arch" type="xml">
+            <tree string="schedules">
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="fsm_schedule_form_view" model="ir.ui.view">
+        <field name="name">Schedules</field>
+        <field name="model">resource.calendar</field>
+        <field name="arch" type="xml">
+            <form string="schedule">
+                <sheet>
+                    <label for="name" class="oe_edit_only"/>
+                    <h1><field name="name"/></h1>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_fsm_schedule" model="ir.actions.act_window">
```

Comment:
@max3903 Can we have to change or update this action? Because it blocks the base flow.

Reply:
Sure